src/types.ts

```diff
@@ -238,3 +238,43 @@ export class InsertOptions {
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface TigrisCollectionType {
```

Comment:
when does ts use .d.ts extension?

Reply:
It is auto generated by typescript. In the next PR, I will be making some changes to make the library a publishable npm package.
 
I will create an issue in Github so we can track steps.

Reply:
sounds good.